Makefile

```diff
@@ -44,6 +44,7 @@ DOCKER_CMD     = docker run --rm -ti -v $(PWD):/go/src/$(SC_PKG) \
 
 ifneq ($(origin DOCKER),undefined)
   # If DOCKER is defined then make it the full docker cmd line we want to use
+  USE_DOCKER=DOCKER=1
```

Comment:
We convert the value of DOCKER from "1" to the actual Docker cmd, and this was my way of saving whether DOCKER was ever set at all.  But now its ugly, so I'm going to change it so that DOCKER is never modified and we'll just define DOCKER_CMD instead.